scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -4,67 +4,59 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
 import "GvsQuickstartHailIntegration.wdl" as QuickstartHailIntegration
 import "GvsUtils.wdl" as Utils
 
-task FilterIntervalListChromosomes {
-    input {
-        File full_interval_list
-        Array[String]+ chromosomes
-    }
-    command <<<
-        # Prepend date, time and pwd to xtrace log entries.
-        PS4='\D{+%F %T} \w $ '
-        set -o errexit -o nounset -o pipefail -o xtrace
-
-        python3 /app/filter_interval_list_chromosomes.py --input-interval-list ~{full_interval_list} \
-            --output-interval-list "filtered.interval_list" --chromosome ~{sep=' --chromosome ' chromosomes}
-    >>>
-    runtime {
-        docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-06-23-alpine"
-    }
-    output {
-        File out = "filtered.interval_list"
-    }
-}
-
 workflow GvsQuickstartIntegration {
     input {
         String branch_name
+        Boolean use_default_dockers = false
         Boolean run_vcf_integration = true
         Boolean run_hail_integration = true
+        Boolean run_exome_integration = true
     }
 
-    File full_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2023-07-19-quicker/"
+    File full_wgs_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+    File full_exome_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/bge_exome_calling_regions.v1.1.interval_list"
```

Comment:
total nit, but it's weird that we call one a wgs interval list and the other one an exome interval list

Reply:
Not sure why that's a problem/nit?